spec/system/admin/enterprise_fees_spec.rb

```diff
@@ -161,6 +161,7 @@
       let!(:fee1) { create(:enterprise_fee, fee_type: "sales", enterprise_id: enterprise.id) }
 
       before do
+        visit admin_enterprise_fees_path
```

Comment:
Loading the page after `fee1` has been created ensures that it's present on the page.